### PR TITLE
Feature/this less renderer

### PR DESCRIPTION
**Pull request by johanneslumpe**
Removed `this` from the renderer so we do not have to bind the render* functions.

### CODE REVIEW

File: modules/renderer/Renderer.js [hunk 1]
```diff
@@ -114,15 +114,15 @@ export default function Renderer(config = { }) {
      * @param {FontFace} fontFace - fontFace which gets rendered
      * @return {string} fontFamily reference
      */
-    renderFont(family, files, properties = { }) {
+    renderFont(family, files, props = { }) {
```
Comment (rofrischmann) [inline modules/renderer/Renderer.js]: I actually used `properties` to show that those are **not** like the other `props` but rather a set of only 5 valid properties that can be set additionally

Comment (johanneslumpe) [inline modules/renderer/Renderer.js]: Ah ok. I can revert that, not a problem - are you cool with the rest of the changes?

Comment (rofrischmann) [inline modules/renderer/Renderer.js]: Yeah just revert that, the rest seems fine :)

**Comment by johanneslumpe** [comment]
@rofrischmann updated!

**Pull request merged**